internal/storage/storage_handle_test.go

```diff
@@ -98,27 +84,27 @@ func (t *StorageHandleTest) TestBucketHandleWhenBucketDoesNotExistWithNonEmptyBi
 }
 
 func (t *StorageHandleTest) TestNewStorageHandleHttp2Disabled() {
-	sc := getDefaultStorageClientConfig() // by default http1 enabled
+	sc := storageutil.GetDefaultStorageClientConfig() // by default http1 enabled
 
 	t.invokeAndVerifyStorageHandle(sc)
 }
 
 func (t *StorageHandleTest) TestNewStorageHandleHttp2Enabled() {
-	sc := getDefaultStorageClientConfig()
+	sc := storageutil.GetDefaultStorageClientConfig()
 	sc.ClientProtocol = mountpkg.HTTP2
 
 	t.invokeAndVerifyStorageHandle(sc)
 }
 
 func (t *StorageHandleTest) TestNewStorageHandleWithZeroMaxConnsPerHost() {
-	sc := getDefaultStorageClientConfig()
+	sc := storageutil.GetDefaultStorageClientConfig()
 	sc.MaxConnsPerHost = 0
 
 	t.invokeAndVerifyStorageHandle(sc)
 }
 
 func (t *StorageHandleTest) TestNewStorageHandleWhenUserAgentIsSet() {
```

Comment:
should we also add test for new fields added in StorageClientConfig?

Reply:
Didn't add, since those are being tested as part of client_test.go.

Reply:
I think no because the StorageHandle is not created there.